### ISSUE TITLE
screen reader doesn't read out the content in chat sessions

### ISSUE THREAD

**Issue by bryanchen-d** [microsoft/vscode#291129]
Testing #290431

I am using NVDA on windows, trying to navigate to the chat window. Some issues
1. I can hit ctrl+alt+i to get the chat input, then navigate back to the chat window using shift+tab
2. but tabbing only stops at the clickable buttons, like the rectangle one in screenshot below, it can read the user prompt but I could not get it to read out all other response output from the model. 

![Image](https://github.com/user-attachments/assets/94bc296d-8080-43d7-be6a-fa4eaf3cfe2d)

**Comment by meganrogge** [comment]
This is what the accessible view is for.
